### DISCUSSION TITLE
Robotidy configuration file that disables long-line-wrapping

### DISCUSSION THREAD

**Discussion by dsidirop** [MarketSquare/robotframework-tidy#234] [General]
I've asked the same question in stackoverflow - feel free to drop a suggestion or two there:

[Stack Overflow](https://stackoverflow.com/questions/70019814/robotidy-configuration-file-that-disables-long-line-wrapping)

**Comment by bhirsz** [comment]
Thanks for the question - I see that I need to improve our documentation a bit (linking etc) since even for me it took few moments to find the information on it :) I've provided answer in the SO.
